.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
something I've seen in templates which I consider useful is to suggest to bug reporters to describe what they expected to happen and then what happened instead.